### DISCUSSION TITLE
string/join

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#212] [Announcements]
# string/join

Java字符串拼接是日常编程中的常见任务。本文详细介绍了Java中不同的字符串拼接方法，包括+号操作符、String.concat、StringBuilder

https://tobebetterjavaer.com/string/join.html

**Comment by shbone** [comment]
dfs

**Comment by Ricehahaa** [comment]
这么多

**Comment by xiaolixaioyu** [comment]
String chenmo = "沉默";
String wanger = "王二";
StringUtils.join(chenmo, wanger);

**Comment by xiaolixaioyu** [comment]
String chenmo = "沉默";
String wanger = "王二";
StringUtils.join(chenmo, wanger);这个方法我看这里的第一个参数应该是集合或者数组啊

**Reply by heymalo** [reply]
传入可变参数，编译器自动优化为数组

**Comment by Git-jimmyLee-hub** [comment]
concat() 源码中 应该是str.getChars(0, otherLen, buf, len); 少写了两个参数
